static/styles/zulip.scss

```diff
@@ -1560,7 +1560,7 @@ a:hover code {
         margin: 5px 0px;
         border: none;
         border-left: 3px solid hsl(0, 0%, 93%);
-        height: 80px;
+        min-height: 80px;
```

Comment:
So, we are generally pretty careful to set a fixed height (100px) for our inline image previews, because that's critical to the scrolling UX.  Without a fixed height for areas containing external resources, the browser cannot lay out the message list exactly as it will be after external resources load, which creates bouncing of the message list as images load.

I suspect the same issue would apply here...

Reply:
Yeah, that makes sense. I'll look into the UI aspects of it, after fixing the backend parts of this PR.

Reply:
I've updated this to use the height provided by the oEmbed provider for the height of the embed div.